From 360cde56ec4852bcde722f80d00e9c0a6cb1e860 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:19 -0400
Subject: [redhat] kabi: Adding symbol find_next_bit (lib/find_next_bit.c)

Message-id: <f972c3ac2514e03389f074610c808be18c0ada84.1410510773.git.poros@redhat.com>
Patchwork-id: 95958
O-Subject: [PATCH RHEL7.1 079/233] kabi: Adding symbol find_next_bit (lib/find_next_bit.c)
Bugzilla: 1062361 1071098 1077335
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1077335,1071098,1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Bitmap helper function for finding next set bit in bitmap.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/find_next_bit
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/find_next_bit

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/find_next_bit b/redhat/kabi/kabi-module/kabi_ppc64/find_next_bit
new file mode 100644
index 0000000..8330bd2
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/find_next_bit
@@ -0,0 +1,2 @@
+#1-
+0xc0a3d105 find_next_bit vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/find_next_bit b/redhat/kabi/kabi-module/kabi_x86_64/find_next_bit
new file mode 100644
index 0000000..8330bd2
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/find_next_bit
@@ -0,0 +1,2 @@
+#1-
+0xc0a3d105 find_next_bit vmlinux EXPORT_SYMBOL
-- 
1.7.1